spec/models/article_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  feature 'admin creates an article' do
+
+    scenario 'they see the article on the page' do
+      visit new_admin_article_path
+    end
+
+  subject {described_class.new(title: "Anything", text: "Lorem ipsum")}
```

Comment:
This wants to be indented one more tab.  I'd also make it a `do` `end` block

Reply:
I'd perhaps replace the `described_class` with build(:article) using factory bot too